packages/docs/src/app/_components/layout/HeaderNavigation/HeaderNavigation.module.scss

```
@@ -0,0 +1,3 @@
+.navigation ul {
+  flex-direction: row;
```

```suggestion
  flex-direction: row;
  gap: var(--navigation--item-to-item-spacing)
```
Bei mir wird sonst die outline des buttons beim hover abgeschnitten, wenn ich auf "Components" bin und über "Getting started" hover

---

das wird oben eh nochmal ne andere Navigation, das ist nur vorübergehend und wird dann richtig gestyled wenn da die richtige Navigation drin ist